docs/audit-log-filter/manage-audit-log-filter.md

```diff
@@ -0,0 +1,11 @@
+# Manage the Audit Log Filter files
```

Comment:
Will we extend this with some examples or explanation of how manual log rotation can be done?

Reply:
Updated.